src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -133,9 +126,6 @@ private IReadOnlyList<ProjectRuleSetConflict> FindConflicts(RuleSetInformation[]
             var ruleSetInfoProvider = this.serviceProvider.GetService<ISolutionRuleSetsInformationProvider>();
             ruleSetInfoProvider.AssertLocalServiceIsNotNull();
 
-            var fileSystem = this.serviceProvider.GetService<IFileSystem>();
```

Comment:
So this would originally return SourceControlledFileSystem - but since the actual usage is only File.Exists we can use FileSystem.